README.md

```diff
@@ -5052,6 +5056,18 @@ Determines whether or not the config should be present on the device. Valid valu
 ##### `description`
 Description of the NVE interface.  Valid values are string, or keyword 'default'.
 
+##### `global_ingress_replication_bgp`
+Configures ingress replication protocol to bgp for all VNIs. Valid values are true, false or keyword 'default'.

```

Comment:
Instead of `Configures` I would use `Sets ingress...`

Reply:
Done